recipe_scrapers/matprat.py

```diff
@@ -1,28 +1,24 @@
+from typing import List, Optional
+
 from ._abstract import AbstractScraper
-from ._utils import get_minutes, normalize_string, get_yields
+from ._utils import get_minutes, get_yields, normalize_string
 
 
 class Matprat(AbstractScraper):
     @classmethod
-    def host(cls):
+    def host(cls) -> str:
         return "matprat.no"
 
-    def title(self):
+    def title(self) -> Optional[str]:
         return self.soup.find("h1").get_text().strip()
 
-    def total_time(self):
-        total_time = 0
+    def total_time(self) -> Optional[int]:
         tt = self.soup.find("span", {"data-epi-property-name": "RecipeTime"})
         if tt:
-            tt1 = normalize_string(tt.get_text())
-            tt2 = get_minutes(tt1)
-            if tt1 and (tt2 == 0):
-                total_time = tt1
-            else:
-                total_time = tt2
-        return total_time
+            tt = get_minutes(normalize_string(tt.get_text()))
+        return tt
```

Comment:
Looks like a good little cleanup :+1:

Reply:
I think this pattern exists for a couple of the scrapers we have. If we can get that scraper health dashboard up and going I think we'll be able to quickly catch complex code like this.

It turns out that the schema code is very `mypy` friendly, but other scrapers are less likely to be.